readme.md

```diff
@@ -70,6 +74,7 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** Describe what the above code is adding to your schema.
 
 * **Your Answer:**
+It grabs the primary key.
```

Comment:
In this case it's setting up a requirement that you insert a Mongo ObjectID when creating a new document.